src/render/PointArray.cpp

```diff
@@ -154,8 +154,8 @@ static OctreeNode* makeTree(int depth, size_t* inds,
         return node;
     }
     // Partition points into the 8 child nodes
-    size_t* childRanges[9];
-    multi_partition(beginPtr, endPtr, OctreeChildIdx(P, center), childRanges+1, 8);
+    std::vector<size_t *> childRanges(9, NULL);
```

Comment:
Yes, that would work.  `std::array` is an option too, assuming C++11

```
std::array<size_t *, 9> childRanges;
childRanges.fill(NULL);
````

which is arguably a bit clearer than the alternatives.